docs/source/intro/configure.rst

```diff
@@ -105,8 +105,21 @@ to match the first two numbers from your version.)
 Configuring the ``DATABASES`` setting
 =====================================
 
-After you've set up a project, configure Django's :setting:`DATABASES` setting
-similar to this::
+After you've set up a project, configure Django's :setting:`DATABASES` setting.
+
+If you have a connection string, you can provide it like this::
+
+    DATABASES = {
+        "default": {
+            "ENGINE": "django_mongodb_backend",
+            "HOST": "mongodb+srv://my_user:my_password@cluster0.example.mongodb.net/myDatabase?retryWrites=true&w=majority&tls=false",
+            "NAME": "my_database",
```

Comment:
Can we support the case in which the database name is in the URI such that `NAME` is not required here? With `parse_uri` and a URI that includes the database name in it, we set `NAME` using the database name in the URI IIRC. If we can't support this, it's probably OK just wanted to make a note of it.

Reply:
I mentioned this case in the design doc. I believe it's best to have the user specify the database rather than implicitly fall back to the "defaultauthdb" in the connection string. For one thing, I think it would be tricky to implement.